tests/grid/grid_in_gmsh_03.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::Mesh info:
+DEAL:: dimension: 2
+DEAL:: no. of cells: 984
```

Comment:
Does the mesh need to be this big? I am sure you can reproduce this with a handful of cells, right?